tensorflow_datasets/core/lazy_imports_lib.py

```diff
@@ -46,6 +82,36 @@ class LazyImporter(object):
   lazily imported here.
   """
 
+  PATH_TRIGGER = 'FAKE_PATH_TRIGGER'
+
+  def __init__(self, path_entry=None):
+    return
+
+  def find_module(self, fullname, path=None):
+    # Accept if fullname is present in _ALLOWED_LAZY_DEPS 
+    if fullname in _ALLOWED_LAZY_DEPS:
+      return self
+    return None
+
+  def load_module(self,fullname):
+    if fullname not in sys.modules:
+      mod = imp.new_module(fullname) # TODO: Give FakeModule(fullname) instead
+      mod.__loader__ = self
+      sys.modules[fullname] = mod
+    return sys.modules[fullname]
```

Comment:
@Conchylicultor Most of the part of the PR is completed.
However, I am running into infinite recursion when I try to load `FakeModule(fullname)` instead of a `imp.new_module(fullname)`.
Could you please guide me through this issue.

Thank you

Reply:
Issue resolved